config/locales/client.en.yml

```diff
@@ -1667,6 +1667,9 @@ en:
         one: "Select at least {{count}} item."
         other: "Select at least {{count}} items."
       invalid_selection_length: "Selection must be at least {{count}} characters."
+      components:
+        categories_admin_dropdown:
+          title: "Categories admin dropdown"
```

Comment:
Was there a particular reason for adding this title? Would something break if we remove it?
It seems to confuse translators and personally I don't see a value in it. I still have to click on the button to find out what it's doing. If we need it, maybe "manage categories" or something like that would be more helpful?

Reply:
100% open to a better copy :+1: